packages/camera/camera_android_camerax/android/src/main/java/io/flutter/plugins/camerax/GeneratedCameraXLibrary.java

```diff
@@ -25,6 +25,82 @@
 @SuppressWarnings({"unused", "unchecked", "CodeBlock2Expr", "RedundantSuppression"})
```

Comment:
Non blocking on this cl. Try to supress warnings at the most minimal level possible with a comment explaining why the warning does not apply. This helps people understand the code and can help in the future to know if it is time to remove the suppression.

Reply:
https://github.com/flutter/flutter/issues/119934